### PR TITLE
Monkey Polymorph Wand

### PR DESCRIPTION
**Pull request by Pangogie**
+ Monkey polymorph wand.
+ Skeleton polymorph. (No wand.)

:cl: Pancake
- add: Beware the monkey wizard.

### CODE REVIEW

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -308,6 +308,9 @@
 - type: Tag
   id: MicrowaveSelfUnsafe
 
+- type: Tag
+  id: Monkey
+
```
Comment (mirrorcult) [inline Resources/Prototypes/tags.yml:313]: do monkeys actually have this tag

Comment (Pangogie) [reply Resources/Prototypes/tags.yml:313]: Nope! I should have probably added that.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/magic.yml [hunk 2]
```diff
@@ -36,16 +46,28 @@
...
+  noSpawn: true
+  components:
+  - type: PolymorphOnCollide
+    polymorph: WizardForcedMonkey
+    whitelist:
+      components:
+      - Body
+    blacklist:
+      tags:
+      - Monkey # can't re-polymorph
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/magic.yml:64]: i could shrimply add a check for recursive polymorphs

**Comment by keronshb** [comment]
Not to popcorn but there was a generalized polymorph staff in SS13 for a random transformation. Shouldn't this be consolidated into a random transformation?

**Comment by Pangogie** [comment]
> Not to popcorn but there was a generalized polymorph staff in SS13 for a random transformation. Shouldn't this be consolidated into a random transformation?

Monkey wizard. Very important.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/magic.yml (hunk 2 above)
Comment (EmoGarbage404) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/magic.yml:64]: shrimply been added.

**Comment by mirrorcult** [comment]
> Not to popcorn but there was a generalized polymorph staff in SS13 for a random transformation. Shouldn't this be consolidated into a random transformation?

that was the staff of change, which i'm implementing later as a wabbajack effect

**Comment by Pangogie** [comment]
Shrimply removed the blacklist tags.

**Comment by Pangogie** [comment]
Requires #9845 btw because of the blacklist removal.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**